CONTRIBUTING.md

```diff
@@ -97,6 +97,7 @@ git push origin master
 ### CHAOSS
 - [Website](https://chaoss.community/)
 - [Get Involved](https://chaoss.community/participate)
+- [Join the CHAOSS Slack](https://chaoss.community/kb-getting-started/) - Join the `#wg-augur-8knot` channel to participate in discussions, meetings, and planning
```

Comment:
While this is a good start, I was thinking it may be better to also mention it in a more paragraph-like format closer to the top of the file so that new contributors see it as one of the first things if they decide to read this file. That way, if they get stuck, they will already be in a position to ask for help in the right place.